### ISSUE TITLE
https://docs.angularjs.org/api/ is DOWN!!! URGENT!!!!!!!!!!!!!!!

### ISSUE THREAD

**Issue by mescalito** [angular/angular.js#13476]
https://docs.angularjs.org/api/ is DOWN!!! URGENT! FIX IT!!!!

**Comment by aciccarello** [comment]
I confirm. Looks like an issue with the imports. The cdn links have null for what I think is the version number.

**Comment by pkozlowski-opensource** [comment]
Duplicate of https://github.com/angular/angular.js/issues/13475

**Comment by petebacondarwin** [comment]
The docs are fixed

**Comment by mescalito** [comment]
what was the problem? if I may ask...

**Comment by petebacondarwin** [comment]
When we move from normal version numbering to beta or to release candidate, the semver doesn't work correctly. In other words, 1.5.x is not compatible with 1.5.0-beta.x.
The way that we choose which CDN version to link the docs site to, is to start with the most recent version and work backwards.
Today, the most recent version is 1.5.0-rc.0, which is not yet on the CDN, but our algorithm is not clever enough to say, "oh, you need to drop back to the beta releases". Instead, it just returns `null`.
Obviously this breaks the docs app. The fix is that we have a custom property on `package.json` (`branchVersion`) which we use to tell us which version to aim for in the docs app.
Stupidly I updated this to 1.5.0-rc.0, before the CDN was ready.
I have no reverted this to 1.5.0-beta.2 for now and will be bump it up again once the CDN is ready.
Later when we go to 1.5.0 proper, we will need to change this value again.

**Comment by razlani** [comment]
@petebacondarwin Thanks for the explanation - it seems to be happening again right now hence the necro post.

**Comment by petebacondarwin** [comment]
@razlani - I believe it is up again?

**Comment by ccarvalhalAtGoContact** [comment]
At this moment, it is down.

**Comment by Narretz** [comment]
@ccarvalhalAtGoContact Please don't post in closed issues about past events. Here's the current issue: https://github.com/angular/angular.js/issues/16205
